contracts/mocks/Proofs.sol

```diff
@@ -69,61 +86,47 @@ contract Proofs is ChainlinkClient, ConfirmedOwner {
      */
     function withdrawLink() public onlyOwner {
         LinkTokenInterface link = LinkTokenInterface(chainlinkTokenAddress());
-        require(link.transfer(msg.sender, link.balanceOf(address(this))), "Unable to transfer");
+        link.transfer(msg.sender, link.balanceOf(address(this)));
```

Comment:
where is the link coming from again? are we just throwing it in the treasury and praying? how do we make sure that people who may decide to collateralize in link aren't gonna have this eat their funds?

Reply:
We chuck link into Escrow.sol for executing jobs.

Reply:
Re: collateralizing in LINK, good point, will modify to make sure people can't deposit in LINK.